api/app/routes/routes.py

```
@@ -21,6 +21,9 @@
 from app.utils.session import SESSIONS
 from fastapi import Body    
 import json
+from fastapi.responses import FileResponse
+from huggingface_hub import hf_hub_download
```

Ruff (0.14.2)

Redefinition of unused `hf_hub_download` from line 5

Remove definition: `hf_hub_download`

(F811)